index.js

```diff
@@ -261,16 +278,22 @@ class AntiSpam extends EventEmitter {
 				return false;
 			}
 			try {
-				if(user.roles.has(role.id)) return;
-				await message.member.addRole(role.id, "Spamming");
+				if(message.member.roles.cache.has(role.id)) return;
+				await message.member.roles.add(role, "Spamming");
 				if (options.muteMessage)
 					await message.channel.send(format(options.muteMessage, message)).catch(e => {
 						if (options.verbose) console.error(e);
 					});
+				if(options.modlogEnabled)
+					var modlogCh = message.guild.channels.cache.find(ch => ch.name == options.modLogChannel)
+					if(modlogCh){
+						modlogCh.send(`Spam detected: ${message.author} got **muted**`)
+					}
 				this.emit('muteAdd', message.member);
 				return true;
 			} catch (error) {
-				const emitted = this.emit('error', message, error, 'mute');
+				console.log(error)
+				// const emitted = this.emit('error', message, error, 'mute');
```

Comment:
I don't think this should be modified

Reply:
Oh wait
That was just because we ere testing lol
.